src/renderer/Layout.tsx

```diff
@@ -102,9 +102,28 @@ export default function Layout() {
   const [state, setState] = React.useState({
     categoryIndex: 0,
     videoIndex: 0,
-    videoState: ipc.sendSync('getVideoState', categories)
+    videoState: ipc.sendSync('getVideoState', categories),
+    videoMuted: false,
+    videoVolume: 1, // (Double) 0.00 - 1.00
   });
 
+  const getVideoPlayer = () => {
+    return (document.getElementById('video-player') as HTMLMediaElement);
+  }
+
+  const updateCurrentVideoSettings = () => {
```

Comment:
Function name isn't totally clear. Sounds like it's updating a video element but actually it's changing the state. Can you change it or add a comment?